build-support/scripts/proto-gen.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
```

Comment:
did you mean to pick one of either proto-gen-entry or proto-gen and retain a single file?

Reply:
Oops yeah. I meant to remove proto-gen-entry 👍